### PR TITLE
Update CHANGELOG.md

### PR DESCRIPTION
**Pull request by overvenus**

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @BusyJay @Hoverbear

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,19 @@
 All notable changes to this project are documented in this file.
 See also [TiDB Changelog](https://github.com/pingcap/tidb/blob/master/CHANGELOG.md) and [PD Changelog](https://github.com/pingcap/pd/blob/master/CHANGELOG.md).
 
+## [2.0.1]
+### Performance
+* Reduce thread_yield call
```
Comment (Hoverbear) [inline CHANGELOG.md]: Reduce number of calls, or the cost of the call?

**Comment by Hoverbear** [comment]
Is https://github.com/pingcap/tikv/pull/3010 part of this release?

**Comment by siddontang** [comment]
No, I think we can take it at 2.0.2

File: CHANGELOG.md (hunk 1 above)
Comment (overvenus) [reply CHANGELOG.md]: Reduce number of calls.

**Pull request merged**